### PR TITLE
Heartdiabetes

### PR DESCRIPTION
**Pull request by adhirpotdarbito**
Added validation for heart age and put balancer parameter for MEN & WOMEN while calculating cvd risk. 
 <div id='description'>
<h3>Summary by Bito</h3>
The changes refactor CVD risk calculation functions, separating logic for men and women. New balancer constants are introduced, and a 'validate_heart_age' function is added to handle edge cases. Comments are improved for clarity.
<br>
<br>
<b>Code change type</b>: Refactoring, Code Structure, Documentation
<br>
<br>
<b>Unit tests added</b>: False
<br>
<br>
<b>Estimated effort to review (1-5, lower is better)</b>: 2
</div>

### CODE REVIEW

**Comment by adhirpotdarbito** [comment]
<div>
<h2>Code Review Agent Run #42ddb1</h2>
<div>
<ul>

<li>
<b>AI Based Review: </b><small>✔️</small>&nbsp;Successful
</li>

<li>
<b>Static Analysis: </b><small>✔️</small>&nbsp;Successful
</li>

</ul>
</div>
<div>
<h3>High-level Feedback</h3>
The refactoring improves code structure by adhering to the Single Responsibility Principle. Continue to review and refactor other functions for similar improvements. Ensure consistent code style and documentation across all files. Consider adding unit tests to validate the new functions and edge case handling. Review the impact of the new balancer constants on the overall risk calculations to ensure accuracy.
</div>
<br>

<div>
<h3>Actionable Issues</h3>

<div id="issue">
<div>
:page_facing_up: src/main/python/__cvd_risk_bmi.py
</div>
<details>
<summary>
Issues: Total - 1, High importance - 0
</summary>
<div>


<table width="100%">
<tbody>

<tr>
<td width="30%">
<a href ="https://github.com/adhirpotdarbito/pranahealth/pull/2/files#r1726398225">Line 110-110</a>
</td>
<td width="70%">
 <small>🟠</small> Medium importance - 1 &nbsp;&nbsp; 
</td>
</tr>

</tbody>
</table>

</div>
</details>
</div>

</div>




<span><i>AI Code Review powered by</i> <sub><sub><a href="https://bito.ai/" target="_blank">Bito Logo</a></sub></sub></span>

</div>

File: src/main/python/__cvd_risk_bmi.py [hunk 1]
```diff
@@ -91,17 +94,27 @@ def heart_age_by_cvd_bmi(cvd,age,sex):
...
     else
-        X = BETA_WOMEN[1]*np.log(22) - CONST_WOMEN
+        X = BETA_WOMEN[1]*np.log(22) - CONST_WOMEN + CONST_WOMEN_BALANCER
         exp = np.exp(X)
         ln_cvd = np.log(1-cvd)/np.log(S_WOMEN)
         result = np.exp(((np.log(ln_cvd/exp))*(1/BETA_WOMEN[0])))
         return np.round(result)
 
+def validate_heart_age(cvd,age,sex):
+    #handle the condition wheere cvd is None
```
Comment (adhirpotdarbito) [inline src/main/python/__cvd_risk_bmi.py:110]: <div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/adhirpotdarbito/pranahealth/pull/2#issuecomment-2303854014>#42ddb1</a><span> - 08/22/2024, 06:07 am</span>
<br>

<div><small>🟠</small> Medium importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The comment contains a typo and doesn't follow proper capitalization.
</div>
<div id="fix">
<b>Fix:</b> Correct the typo in the comment and capitalize the first word to follow Python style guidelines.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -110 +110 @@
 -    #handle the condition wheere cvd is None
 +    # Handle the condition where cvd is None
```
</div>
</details>
</div>
<br>


</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [x] it was incorrectly flagged
